src/store/selectors.ts

```diff
@@ -0,0 +1,11 @@
+import { RootState } from './store'
+
+// export const userSelector = createSelector(
+//   (state: RootState) => state.user.value,
+//   (value) =>
+//     value.find(({ products }: Order) => products.find((id) => id))?.title ??
+//     'none',
+// )
+
+export const getUserDataSelector = (state: RootState) => state.user.value
```

Comment:
Selector is better to name like selectBlaBlaSomething

Reply:
fixed